lib/transbank/sdk/patpass/patpass_by_webpay/transaction.rb

```diff
@@ -0,0 +1,73 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      class Transaction
+        extend Transbank::Utils::NetHelper
+        CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+
+        class << self
+
+          def create(buy_order:, session_id:, amount:, return_url:, options: nil)
```

Comment:
I think this is missing the 'details' property

Reply:
here is missing all of the wpm_details array params, please review the documentation 
![image](https://user-images.githubusercontent.com/522832/62655687-85733700-b930-11e9-800d-b4d32bfe02f9.png)